package.json

```
@@ -36,7 +36,7 @@
   "bugs": "https://github.com/EmanuelAmico/staffys-api/issues",
   "scripts": {
     "start": "node dist/index.js",
-    "dev": "tsc-watch --onSuccess \"node dist/index.js\"",
+    "dev": "tsc-watch --onSuccess \"node dist/src/index.js\"",
```

Esto es por docker Rodri, hay que dejarlo así cómo estaba. No levantes la api con `npm run dev` sino con `docker compose up local`.
```suggestion
    "dev": "tsc-watch --onSuccess \"node dist/index.js\"",
```